.gitignore

```diff
@@ -1,24 +1,27 @@
-# Python byte-compiled, optimized and DLL files

```

Comment:
Yeah, looks like DOS line endings. I switched it to UNIX. Will this be an issue?